--- syslog-ng-1.6.8/configure.in.orig	2005-05-26 10:33:54.000000000 +0200
+++ syslog-ng-1.6.8/configure.in	2005-05-26 19:08:55.827602272 +0200
@@ -170,7 +170,7 @@
 dnl Checks for library functions.
 AC_CHECK_LIB(door, door_create)
 AC_CHECK_LIB(socket, socket)
-AC_CHECK_LIB(nsl, gethostbyname)
+AC_SEARCH_LIBS(gethostbyname, nsl)
 AC_CHECK_LIB(regex, regexec)
 
 AC_CHECK_FUNCS(strdup strtol inet_aton inet_ntoa getopt_long)
